### PR TITLE
Problem: Swagger UI does not include NFT APIs

### LINKED ISSUE #531: Problem: Swagger UI does not include NFT APIs
The swagger UI is using the pre-built one from Cosmos
https://github.com/crypto-org-chain/chain-main/blob/cc09e637f86b18ffa3b49b0cca63fa6129be2658/app/app.go

Hence, it does not include the NFT endpoints.

It would be good if the project maintain its own Swagger file and able to re-generate it when new features are added.

### PR DESCRIPTION
**Pull request by yihuang**
fixes #531

Solution:
- vendor in swagger script and update the assets
- port protobuf tools from cosmos-sdk
- update swagger title

### How to use
- `make proto-swagger-gen` update swagger files
- `make update-swagger-docs` update statik embed swagger assets
- `make proto-all` format/lint protobuf files and gen golang code

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

**Review by devashishdxt — APPROVED** [review]

File: app/docs/config.json [hunk 1]
```diff
@@ -0,0 +1,122 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cosmos SDK - Legacy REST and gRPC Gateway docs",
```
Comment (calvinaco) [inline app/docs/config.json:4]: We can also update the swagger title to Crypto.org Chain?
```suggestion
    "title": "Crypto.org Chain - Legacy REST and gRPC Gateway docs",
```

Comment (calvinaco) [reply app/docs/config.json:4]: and this will be reflected on the generated swagger UI right?

Comment (yihuang) [reply app/docs/config.json]: yeah, seems to be the case

File: app/docs/statik/init.go [hunk 2]
```diff
@@ -0,0 +1,3 @@
+package statik
+
```
Comment (tomtau) [inline app/docs/statik/init.go:2]: lol statik.go file has 1.6MB

**Review by tomtau — COMMENTED** [review]
@yihuang one thing I remember I think @cdc-Hitesh mentioned was that Swagger examples used a wrong bech32 address hrp -- can this be fixed too?

File: app/docs/statik/init.go (hunk 2 above)
Comment (yihuang) [reply app/docs/statik/init.go:2]: yeah, it embed the whole `app/docs/swagger-ui` directory.

**Review by leejw51crypto — APPROVED** [review]
lgtm

**Comment by yihuang** [comment]
> @yihuang one thing I remember I think @cdc-Hitesh mentioned was that Swagger examples used a wrong bech32 address hrp -- can this be fixed too?

don't know which one it is, if it's cosmos-sdk's api, the doc is probably written in cosmos-sdk source code, then there's not much we can do.

**Comment by tomtau** [comment]
> > @yihuang one thing I remember I think @cdc-Hitesh mentioned was that Swagger examples used a wrong bech32 address hrp -- can this be fixed too?
> 
> there are many `cosmos` appeared in the `swagger.yaml` file, do we really want to patch those? 😄

There's no problem with the protobuf types etc. -- but for examples payloads... if one tries to use the example payload, it'll fail with an error that the address validation failed. Maybe can open an issue for that

**Comment by yihuang** [comment]
> > > @yihuang one thing I remember I think @cdc-Hitesh mentioned was that Swagger examples used a wrong bech32 address hrp -- can this be fixed too?
> > 
> > 
> > there are many `cosmos` appeared in the `swagger.yaml` file, do we really want to patch those? 😄
> 
> There's no problem with the protobuf types etc. -- but for examples payloads... if one tries to use the example payload, it'll fail with an error that the address validation failed. Maybe can open an issue for that

done, seems manageable.

**Review by tomtau — APPROVED** [review]

**Pull request merged**